packages/flutter/test/material/expansion_tile_test.dart

```diff
@@ -70,6 +70,7 @@ void main() {
                 key: expandedKey,
                 initiallyExpanded: true,
                 title: const Text('Expanded'),
+                subtitle: const Text('Subtitle'),
```

Comment:
I don't think you should add this line here. We should assume that the default `ExpansionTile` does not have its subtitle defined. 

Instead, we should write a new test that validates that if ExpansionTile.subtitle is defined, it is indeed shown.

Reply:
It seems that this line is also breaking the presubmit tests.